pkg/storaged/client.js

```diff
@@ -106,37 +120,40 @@ define([
          * efficient since it reduces the number of D-Bus calls done
          * by the cache.
          */
-        return client.storaged_client.proxies(iface, "/org/storaged/Storaged",
+        return client.storaged_client.proxies(STORAGED_IFACE_PFX + "." + iface,
```

Comment:
I wonder if we should rename the function to something like storage_proxies or udisks_proxies if we add a prefix explicitly. Or is the context (file in storage package) enough to make the code readable?